src/content/blog/astro-4120.mdx

```diff
@@ -0,0 +1,140 @@
+
+### Performance
+
+To test the viability of this idea we built [server-islands.com](https://server-islands.com/). This demo is inspired by the [Next.js Partial Prerendering](https://www.partialprerendering.com/) demo, reskinned to suit Astro's personality. **Server Islands** has a lot in common with Partial Prerendering, especially in its goal of allowing increased cacheability of pages.
+
+| <span class="whitespace-nowrap">Metrics</span>                                                                                                 | Partial Prerendering                                                 | Server Islands                                                       |
+| ---------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------- | -------------------------------------------------------------------- |
+| <abbr class="sm:hidden no-underline" title="Time To First Byte">TTFB</abbr><span class="hidden sm:inline">Time To First Byte</span>            | 0.838s                                                               | 0.766s                                                               |
+| <abbr class="sm:hidden no-underline" title="First Contentful Paint">FCP</abbr><span class="hidden sm:inline">First Contentful Paint</span>     | 1.740s                                                               | 1.251s                                                               |
+| <abbr class="sm:hidden no-underline" title="Largest Contentful Paint">LCP</abbr><span class="hidden sm:inline">Largest Contentful Paint</span> | 1.740s                                                               | 1.405s                                                               |
+| <abbr class="sm:hidden no-underline" title="Total Blocking Time">TBT</abbr><span class="hidden sm:inline">Total Blocking Time</span>           | 0.093s                                                               | 0.018s                                                               |
+| Source                                                                                                                                         | [WebPageTest](https://www.webpagetest.org/result/240715_BiDc15_G7W/) | [WebPageTest](https://www.webpagetest.org/result/240716_AiDc6D_D1N/) |
+
+As you can see from this table, the numbers are extremely close. Server Islands has an advantage in painting slightly faster due to being more lightweight both in the infrastructure setting in front of the request, and the amount of code needed in the client throughout the process. Partial prerendering has an advantage in that it can finish slightly faster, due to starting the request for its deferred content on the server before the HTML reaches the user's browser.
```

Comment:
Is the table data right? Feels like SI is faster across the board, 1.251 vs. 1.740 is a 30% speed up. Doesn't match the "extremely close" message at all IMO.

If we're faster, lets celebrate it! You can still couch and caveat but this is the part where we should be bullish and advocate that our approach is superior perf-wise (and why).

Reply:
Good point, reworked this section to be less couch-ey, and moved portability into its own section.